web/marketing/css/style.css

```diff
@@ -38,4 +129,49 @@ body{
 
 .toggler:hover{
     cursor:pointer;
+}
+
+#foModalLabel, 
+#roModalLabel{
```

Comment:
What's `fo` and `ro`? Any reason these names have different conventions than rest of the names?

Reply:
fo is free offer and ro is referral offer. I tried to keep the names short.

Reply:
I would go with `#free-offer-modal-label` or `#free-offer-label` or something like that. Previously when looking at the filenames I got confused about `fo` and `ro` as well.

Reply:
Alternatively shorten the part that people are more used to e.g. `lbl`, but I really don't see a good reason to shorten this.

Reply:
OK, I'll update the filenames and id's.